inloop/templates/admin/tasks/plagiarism.html

```diff
@@ -0,0 +1,24 @@
+{% extends 'admin/change_list.html' %} {% load i18n %}
+{% load staticfiles %}
+
+{% block object-tools %}
+
+    {% if test %}
+
+    <div style="margin-bottom: 75px; background-color: #79aec8; color: #fff; padding: 20px; border-radius: 20px; margin-right: 20px;">
+        Last plagiarism test: {{ test.created_at }} ({{ test.time_description }} ago) -
```

Comment:
Style: with the `timesince` filter/tag, it would look like `({{ test.created_at|timesince }} ago)`.